views/mrp_production_view.xml

```diff
@@ -1,41 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Same as python files, we split the xml files based on the model of view we inherit/create